Makefile

```diff
@@ -1,33 +1,99 @@
-include $(DEVKITARM)/base_tools
+# GBA rom header
+TITLE       := AGB KIRBY AM
+GAME_CODE   := B8KE
+MAKER_CODE  := 01
+REVISION    := 0
+GAME_LANGUAGE := ENGLISH
+KEEP_TEMPS  ?= 0
```

Comment:
How is this supposed to work? I tested with `make compare -j KEEP_TEMPS=1` and the `.i` and `.s` files are directly flooding into ROOT_DIR.

In addition, this is not used for the subdirectory targets.

Reply:
Quite honestly, I'm not sure why it is the way it is. Flooding root is apparently the expected behavior (tested on pokeemerald) but I agree it's a bit ugly to do. I changed KEEP_TEMPS to have the original behavior of this project.